test/fixtures/syntax_error_in_tests/input/two_fer_test.rb

```diff
@@ -0,0 +1,3 @@
+require 'minitest/autorun'
+
+,,,,,,,
```

Comment:
This could be clearer.

`''This is purposefully a syntax error`

I had to count the pairs and make sure there was an odd number here.  3 is easier to see, 1 is easiest, I think, but a single quote with the statement that it is purposefully a syntax error is something I wouldn't miss.

- - -

For some reason I missed the PR review stage... ;(

Reply:
Wait those are commas... hard to see that. Perhaps we should raise the error we are testing for?  `raise SyntaxError` is probably even clearer?

Reply:
If you want to PR `'This is purposefully a syntax error` then I'll happily merge that. I'd rather have an actual syntax error, than manually raise one.

> For some reason I missed the PR review stage... ;(

There's not a review stage on this atm - I'm pretty much just merging to master, but putting it through PRs for people to get visibility :)